[8.16 changelog] add known issue for `restart_on_cert_change` panic (#46385)
elastic/beats: CHANGELOG.asciidoc

=== Beats version 8.16.6
https://github.com/elastic/beats/compare/v8.16.5\...v8.16.6[View commits]

==== Known issues

> - `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. For more details, refer to <<linux-seccomp>>.

Reviewer: Oh no... I've already forgotten how to write AsciiDoc! 🙈  I can update the other PRs and babysit them to make sure the build succeeds. 

```suggestion
- `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. For more details, refer to {filebeat-ref}/linux-seccomp.html[Use Linux Secure Computing Mode (seccomp)].
```

Author: ok then. I added the known issue for every patch on every minor PR. but the link to `linux-seccomp` is still broken.

if you need help, let me know

> - `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. Check our https://www.elastic.co/docs/reference/beats/filebeat/linux-seccomp[docs] for further details on how to configure a seccomp policy.

Reviewer: Ah we should actually probably use a local link:

```suggestion
- `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. For more details, refer to <<linux-seccomp>>.
```
